src/main/scala/org/broadinstitute/dsde/workbench/sam/openam/PostgresAccessPolicyDAO.scala

```diff
@@ -1174,7 +1174,7 @@ class PostgresAccessPolicyDAO(protected val dbRef: DbReference,
         FullyQualifiedResourceId(
           rs.get[ResourceTypeName](prt.resultName.name),
           rs.get[ResourceId](pr.resultName.name)))
-        .single.apply()
```

Comment:
auto apply of `()` is deprecated. Hence explicitly using `()` for unary methods are preferred